scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -38,21 +39,32 @@ workflow GvsBulkIngestGenomes {
         # End GvsImportGenomes
     }
 
+    parameter_meta {
+        data_table_name: "The name of the data table; This table holds the GVCFs to be ingested; `sample` is the default."
+        sample_id_column_name: "The column that will be used for the sample name / id in GVS; the <data_table_name>_id will be used as the default"
+        vcf_files_column_name: "The column that supplies the path for the GVCFs to be ingested--when not specified, the workflow will attempt to derive the column"
+        vcf_index_files_column_name: "The column that supplies the path for the GVCF index files to be ingested--when not specified, the workflow will attempt to derive the column"
+        sample_set_name: "The recommended way to load samples; Sample sets must be created by the user"
+    }
+
+    ## Start off by getting the Workspace ID to query for more information
     call GetWorkspaceId
 
+    ## Next, use the workspace ID to get the Workspace Name
     call GetWorkspaceName {
         input:
             workspace_id = GetWorkspaceId.workspace_id,
             workspace_bucket = GetWorkspaceId.workspace_bucket,
     }
 
-    call GetColumnNames { ## TODO should we even run this at all if we have values for all 4?
+
+    call GetColumnNames {
         input:
-            workspace_id = GetWorkspaceId.workspace_id,
             workspace_name = GetWorkspaceName.workspace_name,
             workspace_namespace = GetWorkspaceName.workspace_namespace,
-            samples_table_name = samples_table_name,
-            sample_id_column_name = sample_id_column_name,
+            sample_set_name = sample_set_name,
+            data_table_name = data_table_name,
+            user_defined_sample_id_column_name = sample_id_column_name, ## NOTE: the user needs to define this, or it will default to the <entity>_id col
```

Comment:
```suggestion
            user_defined_sample_id_column_name = sample_id_column_name, ## NOTE: the user needs to define this, or it will default to the <entity>_id column
```